README.md

```diff
@@ -44,113 +43,58 @@ Rocket is extensively documented:
 [Guide]: https://rocket.rs/guide/
 [API Documentation]: https://api.rocket.rs
 
-The official community support channels are [`#rocket:mozilla.org`] on Matrix
-and the bridged [`#rocket`] IRC channel on Libera.Chat at `irc.libera.chat`. We
-recommend joining us on [Matrix via Element]. If your prefer IRC, you can join
-via the [Kiwi IRC client] or a client of your own.
+Documentation for the `master` branch is available at https://rocket.rs/master
+and https://api.rocket.rs/master.
 
-[`#rocket:mozilla.org`]: https://chat.mozilla.org/#/room/#rocket:mozilla.org
-[`#rocket`]: https://kiwiirc.com/client/irc.libera.chat/#rocket
-[Matrix via Element]: https://chat.mozilla.org/#/room/#rocket:mozilla.org
-[Kiwi IRC Client]: https://kiwiirc.com/client/irc.libera.chat/#rocket
+Documentation for major release version `${x}` is available at
+`https://[api.]rocket.rs/v${x}`. For example, the v0.4 docs are available at
+https://rocket.rs/v0.4 and https://api.rocket.rs/v0.4.
+
+Finally, API docs for active git branches are available at
+`https://api.rocket.rs/${branch}`. For example, API docs for the `master` branch
+are available at https://api.rocket.rs/master. Branch rustdocs are built and
+deployed on every commit.
 
 ## Examples
 
-An extensive number of examples are provided in the `examples/` directory. Each
-example can be compiled and run with Cargo. For instance, the following sequence
-of commands builds and runs the `Hello, world!` example:
+The [examples](examples#readme) directory contains complete crates that showcase
+Rocket's features and usage. Each example can be compiled and run with Cargo.
+For instance, the following sequence of commands builds and runs the `hello`
+example:
 
 ```sh
 cd examples/hello
 cargo run
 ```
 
-You should see `Hello, world!` by visiting `http://localhost:8000`.
-
-## Building and Testing
-
-The `core` directory contains the three core libraries: `lib`, `codegen`, and
-`http` published as `rocket`, `rocket_codegen` and `rocket_http`, respectively.
-The latter two are implementations details and are reexported from `rocket`.
```

Comment:
I'm not really sure where this fits, and I'm not sure this fits in the contributing guide at all. I feel like this would be best in a completely different type of guide, a development guide of sorts, like [the one for rustc](https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html). But if you can think of a good place for it here, I'm happy to work it in.

Reply:
Yeah, a development guide would be a great place to put this. I just wanted to make sure we weren't just deleting important information for new contributors.